[Gateway] 'gulp build' never exits

##### **Overview of the issue**

Running 'gulp build' on Gateway generated from today master #8c672b3f82238352de3a65bc8a21b8b0224aa476 never exits though build is finsihed.

Env: Windows 7, Node v4.2.4, gulp-cli 1.2.1

~~~
gulp build
[14:09:02] Using gulpfile C:\projets\gateway\gulpfile.js
[14:09:02] Starting 'build'...
[14:09:02] Starting 'clean'...
[14:09:02] Finished 'clean' after 25 ms
[14:09:02] Starting 'copy'...
[14:09:02] Finished 'copy' after 19 ms
[14:09:02] Starting 'wiredep:app'...
[14:09:02] Finished 'wiredep:app' after 123 ms
[14:09:02] Starting 'ngconstant:prod'...
[14:09:02] Finished 'ngconstant:prod' after 72 ms
[14:09:02] Starting 'eslint'...
[14:09:02] gulp-notify: [JHipster Gulp Build] Error: Expected indentation of 0 s
pace characters but found 1.
[14:09:02] Finished 'eslint' after 547 ms
[14:09:02] Starting 'images'...
[14:09:02] Starting 'sass'...
[14:09:02] Tested 3 tests, 3 passes, 0 failures: PASS
[14:09:03] Finished 'sass' after 403 ms
[14:09:03] Starting 'styles'...
[14:09:03] Finished 'styles' after 1.87 ms
[14:09:04] gulp-notify: [JHipster Gulp Build] Error: Expected indentation of 16
space characters but found 14.
[14:09:04] gulp-notify: [JHipster Gulp Build] Error: 'data' is defined but never
 used
[14:09:05] gulp-notify: [JHipster Gulp Build] Error: You should not use directly
 the "undefined" keyword. Prefer angular.isUndefined or angular.isDefined
[14:09:05] gulp-notify: [JHipster Gulp Build] Error: 'account' is defined but ne
ver used
[14:09:05]
src\main\webapp\app\app.constants.js
  11:2  error  Expected indentation of 0 space characters but found 1  indent

src\main\webapp\app\blocks\interceptor\auth.interceptor.js
  23:15  error  Expected indentation of 16 space characters but found 14  indent


src\main\webapp\app\components\alert\alert-error.directive.js
  24:47  error  'data' is defined but never used  no-unused-vars

src\main\webapp\app\services\auth\auth.jwt.service.js
  43:21  error  You should not use directly the "undefined" keyword. Prefer angu
lar.isUndefined or angular.isDefined  angular/definedundefined

src\main\webapp\app\services\auth\auth.service.js
  106:56  error  'account' is defined but never used  no-unused-vars

? 5 problems (5 errors, 0 warnings)

[14:09:05] gulp-imagemin: Minified 3 images (saved 2.55 kB - 6.4%)
[14:09:05] Finished 'images' after 3.03 s
[14:09:05] Starting 'usemin'...
[14:09:20] Finished 'usemin' after 15 s
[14:09:20] Finished 'build' after 18 s
~~~~

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder** 

~~~json
{
  "generator-jhipster": {
    "jhipsterVersion": "2.27.0",
    "baseName": "gateway",
    "packageName": "com.actoll.ecobil.central",
    "packageFolder": "com/actoll/ecobil/central",
    "authenticationType": "jwt",
    "hibernateCache": "ehcache",
    "clusteredHttpSession": "no",
    "websocket": "no",
    "databaseType": "sql",
    "devDatabaseType": "h2Memory",
    "prodDatabaseType": "postgresql",
    "searchEngine": "no",
    "buildTool": "maven",
    "jwtSecretKey": "c3574e02f9bf5a9999fe8dd841f40dac2f0a9b80",
    "rememberMeKey": "1b9aa9ea8e44fabedf7a9999d4507f62b1b7bbac",
    "useSass": true,
    "applicationType": "gateway",
    "testFrameworks": [
      "gatling"
    ],
    "enableTranslation": false,
    "enableSocialSignIn": false
  }
}
~~~

--- a/generators/server/templates/_pom.xml
+++ b/generators/server/templates/_pom.xml
@@ -1053,6 +1053,7 @@
                                 </goals>
                                 <configuration>
                                     <buildTool>gulp</buildTool>
+                                    <buildArgs>build --no-notification</buildArgs>
                                 </configuration>
                             </execution>
                         </executions>

